ci/travis_script_python.sh

```diff
@@ -4,28 +4,19 @@ set -e
 
 PYTHON_DIR=$TRAVIS_BUILD_DIR/python
 
+# Re-use conda installation from C++
+export MINICONDA=$TRAVIS_BUILD_DIR/miniconda
+export PATH="$MINICONDA/bin:$PATH"
+export LD_LIBRARY_PATH="$MINICONDA/lib:$LD_LIBRARY_PATH"
```

Comment:
Is this needed? In theory this should not be necessary (handled by conda's shared library patching)